lib/compilers/java.js

```diff
@@ -103,7 +103,13 @@ export class JavaCompiler extends BaseCompiler {
         ];
     }
 
-    async handleInterpreting(key, executeParameters) {
+    async handleInterpreting(source, executeParameters) {
```

Comment:
I'm trusting you on this; but this looks awfully like what the compiler already does; perhaps modeling the "interpret" feature isn't best here. If it works though :)